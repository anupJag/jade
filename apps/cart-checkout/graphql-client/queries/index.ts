export { QUERY_ALL_SLOTS_INFO_AND_LATEST_SLOT, QUERY_SLOTS_BY_DATERANGE } from './DeliverySlots';
