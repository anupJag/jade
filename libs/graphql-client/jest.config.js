module.exports = {
  name: 'ui-graphql-client',
  preset: '../../../jest.config.js',
  transform: {
    '^.+\\.[tj]sx?$': ['babel-jest', { cwd: __dirname, configFile: './babel-jest.config.json' }],
  },
  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
  coverageDirectory: '../../../coverage/libs/ui/graphql-client',
};
