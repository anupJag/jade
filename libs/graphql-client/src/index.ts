export * from './lib/graphqlClient';
export * from './lib/contentfulGraphqlClient';
export * from './lib/queries';
export * from './lib/authorisedGraphqlClient';
