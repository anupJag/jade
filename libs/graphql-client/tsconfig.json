{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "jsx": "react",
    "allowJs": false,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "resolveJsonModule": true,
    "types": ["node", "jest"]
  },
  "files": [
    "../../../node_modules/@nrwl/react/typings/cssmodule.d.ts",
    "../../../node_modules/@nrwl/react/typings/image.d.ts"
  ],
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
}
