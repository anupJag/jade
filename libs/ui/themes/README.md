# fe-themes

This library was generated with [Nx](https://nx.dev).

## Running unit tests

Run `ng test fe-themes` to execute the unit tests via [Jest](https://jestjs.io).
