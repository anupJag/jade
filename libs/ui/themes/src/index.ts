export * from './lib/defaultTheme';
export * from './lib/styled';
export * from './lib/themeProvider';
export { default as styled } from './lib/styled';
export * from './lib/GlobalStyles';
