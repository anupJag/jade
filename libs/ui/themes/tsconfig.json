{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "jsx": "react",
    "allowJs": false,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "types": ["node", "jest"]
  },
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
}
