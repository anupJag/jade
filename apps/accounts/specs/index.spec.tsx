import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider } from '@jade/ui/themes';
import { defaultTheme } from '@jade/ui/themes';
import { CSSReset } from '@chakra-ui/core';
import { ApolloProvider } from '@apollo/react-hooks';
import { initializeApollo } from '@jade/graphql-client';
import { ApolloClient } from '@apollo/client';
import { NormalizedCacheObject } from 'apollo-cache-inmemory';
import Index from '../pages/index';

describe(' Index', () => {
  it('should render successfully', () => {
    const initApolloClient = initializeApollo();
    const { baseElement } = render(
      <ApolloProvider client={initApolloClient}>
        <ThemeProvider theme={defaultTheme}>
          <CSSReset />
          <Index />
        </ThemeProvider>
      </ApolloProvider>,
    );
    expect(baseElement).toBeTruthy();
  });
  xit('snapshot check', () => {
    const initApolloClient = initializeApollo();
    const { asFragment } = render(
      <ApolloProvider client={initApolloClient}>
        <ThemeProvider theme={defaultTheme}>
          <CSSReset />
          <Index />
        </ThemeProvider>
      </ApolloProvider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
