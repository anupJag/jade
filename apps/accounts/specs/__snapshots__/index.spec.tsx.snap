// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Index snapshot check 1`] = `
<DocumentFragment>
  <div>
    <div
      class="css-1djjrov"
    >
      <a
        class="css-81hfzx"
        href="#main-content"
      >
        skip to main content
      </a>
    </div>
    <header
      class="css-up7qba"
    >
      <div
        class="css-1taa2i3"
      >
        <div
          class="css-lglhnm"
        >
          <div
            class="css-ov1ktg"
          >
            <div>
              <a
                class="css-1iwxjwb"
                title="Go to homepage"
              >
                <svg
                  aria-hidden="true"
                >
                  logo.svg
                </svg>
                <span
                  class="css-1a7a8z8"
                >
                  Go to homepage
                </span>
              </a>
            </div>
          </div>
          <div
            class="css-ov1ktg"
          >
            <nav
              class="css-0"
            >
              <ul
                class="css-1q0n0ua"
              >
                <li
                  class="css-1i54o6h"
                >
                  <a
                    class="css-1evzoma"
                    href="/categories"
                    title="Categories"
                  >
                    Categories 
                    <svg
                      aria-hidden="true"
                      class="css-1im46kq"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                        fill="currentColor"
                      />
                    </svg>
                  </a>
                </li>
                <li
                  class="css-1i54o6h"
                >
                  <a
                    class="css-1evzoma"
                    href="/offers"
                    title="Offers"
                  >
                     
                  </a>
                </li>
                <li
                  class="css-1i54o6h"
                >
                  <a
                    class="css-1evzoma"
                    href="/help/faq"
                    title="Help"
                  >
                    Help 
                  </a>
                </li>
              </ul>
            </nav>
          </div>
          <div
            class="css-3ledld"
          >
            <div
              class="MuiPaper-root makeStyles-root-7 MuiPaper-elevation1 MuiPaper-rounded"
            >
              <div
                aria-expanded="false"
                class="MuiAutocomplete-root MuiAutocomplete-fullWidth"
                role="combobox"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                >
                  <div
                    class="MuiInputBase-root MuiInput-root MuiInput-underline makeStyles-input-10 MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
                  >
                    <input
                      aria-autocomplete="both"
                      aria-invalid="false"
                      aria-label="Search on store"
                      autocapitalize="none"
                      autocomplete="off"
                      class="MuiInputBase-input MuiInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd"
                      id="search"
                      placeholder="Search on store"
                      spellcheck="false"
                      type="text"
                      value=""
                    />
                    <div
                      class="MuiAutocomplete-endAdornment"
                    />
                  </div>
                </div>
              </div>
              <button
                aria-label="search"
                class="MuiButtonBase-root MuiIconButton-root makeStyles-iconButton-8"
                tabindex="0"
                type="submit"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="css-3ledld"
          >
            <div
              class="css-ic3oet"
            >
              <nav
                class="css-0"
              >
                <ul
                  class="css-1q0n0ua"
                >
                  <div
                    class="css-ic3oet"
                  >
                    <a
                      class="css-1evzoma"
                      href="/signin"
                      title="Signin"
                    >
                      Signin 
                    </a>
                  </div>
                  <div
                    class="css-ic3oet"
                  >
                    /
                  </div>
                  <div
                    class="css-ic3oet"
                  >
                    <a
                      class="css-1evzoma"
                      href="/register"
                      title="Register"
                    >
                      Register 
                    </a>
                  </div>
                </ul>
              </nav>
            </div>
          </div>
          <div
            class="css-1oayl83"
          >
            <button
              aria-label="cart"
              class="MuiButtonBase-root MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <span
                  class="MuiBadge-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    style="color: rgb(76, 175, 80);"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                    />
                  </svg>
                  <span
                    class="MuiBadge-badge MuiBadge-anchorOriginTopRightRectangle MuiBadge-colorSecondary MuiBadge-invisible"
                  >
                    0
                  </span>
                </span>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </header>
    <div
      class="css-1mq7rrk"
      id="main-content"
      role="main"
    >
      <div
        class="css-1q9iu3a"
      >
        <div
          class="css-1taa2i3"
        >
          <h1>
            Welcome to Accounts Page!
          </h1>
        </div>
      </div>
    </div>
    <footer />
  </div>
</DocumentFragment>
`;
